### PR TITLE
Basic design of the training program

### PR DESCRIPTION
**Pull request by prafull11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,47 @@
...
-3. Can delete one or all orders of a user at a time
-4. Store the data in any DB of your choice
-5. Containerize your services and test it.
-6. Package your application using Helm and deploy on K8s and test it.
-Eg: http://localhost:80/users -> all users
-    http://localhost:80/users/0/orders -> all orders of user 0
-    http://localhost:80/users/0/orders/0 - order 0 of user 0
+### Introduction
+
+An intense 2 months program for young software engineers with at least 2 years of industry experience. This program will train you to become a cloud native engineer with technologies, like Golang, Kubernetes, microservices, and cloud platforms.
```
Comment (madhurtrilio) [inline README.md:5]: don't specify duration as 2 months. We don't want this to be a time bound exercise. The duration would be set according to how well an individual can pick
Don't specify min experience, can specify areas of knowledge like : Person having worked on any backend language, has some virtualization exposure, knows database relations and understands REST API's has an added advantage

File: README.md [hunk 2]
```diff
@@ -1,40 +1,47 @@
...
+- [Study Material]()
+- [Assignment]()
+---
 
-- Code should be structured even though it is an assignment
+### Docker Training
 
-- write up a simple summary of what you tried to do and how did you do in the commit comments
+- [Course to Undergo]()
+- [Assignments]()
```
Comment (madhurtrilio) [inline README.md:27]: please add [Study Material]()

File: README.md [hunk 3]
```diff
@@ -1,40 +1,47 @@
...
 
-- Follow some good go repo template like https://github.com/IBM/go-repo-template
 ---
 
-### Submission
+### Kubernetes Training
 
-- Fork this repository and work on it
+- [Course to Undergo]()
+- [Assignments]()
```
Comment (madhurtrilio) [inline README.md:34]: please add [Study Material]()

**Review by madhurtrilio — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (madhurvelotio) [reply README.md:28]: Is that resolved?

File: README.md (hunk 3 above)
Comment (madhurvelotio) [reply README.md:35]: Is that resolved?

**Review by madhurvelotio — APPROVED** [review]

**Pull request merged**